crates/sargon/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/simple_non_fungible_resource_bounds.rs

```diff
@@ -0,0 +1,195 @@
+use crate::prelude::*;
+
+/// Represents the bounds for a simple non-fungible resource, which can be either exact or not exact.
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Enum)]
+pub enum SimpleNonFungibleResourceBounds {
+    Exact {
```

Comment:
any doc on variants from RET you can carry over?

Reply:
nope